WickedEngine/ssrCS.hlsl

```diff
@@ -6,6 +6,9 @@ TEXTURE2D(texture_lineardepth_minmax, float2, TEXSLOT_ONDEMAND1);
 
 RWTEXTURE2D(output, float4, 0);
 
+//#define USE_RAYMARCH
+
+#ifdef USE_RAYMARCH
```

Comment:
I tried it, looks amazing. Slower than the old one but much nicer. Can add sliders later to reduce sample count and other params.